install.rdf

```diff
@@ -9,7 +9,7 @@
     <em:homepageURL>https://github.com/bwinton/VerticalTabs</em:homepageURL>
     <em:updateURL>https://people.mozilla.com/~bwinton/TabCenter/tab-center.update.rdf</em:updateURL>
     <em:iconURL>resource://tabcenter/skin/icon.png</em:iconURL>
-    <em:description>A new, vertical, arrangement for your tabs.</em:description>
+    <em:description>A new, vertical, arrangement for your tabs. Learn more about this Test Pilot experiment at https://testpilot.firefox.com/.</em:description>
```

Comment:
I always do it, but html doesn't render them, so it's like I'm just wasting bytes…                                                                
🙂